packages/polling-controller/src/PollingController.test.ts

```diff
@@ -198,18 +215,21 @@ describe('PollingController', () => {
       });
       controller.setIntervalLength(TICK_TIME * 3);
       controller.startPollingByNetworkClientId('mainnet');
-      jest.advanceTimersByTime(TICK_TIME);
+      jest.advanceTimersByTime(0);
       await Promise.resolve();
-      expect(controller._executePoll).not.toHaveBeenCalled();
+      expect(controller._executePoll).toHaveBeenCalledTimes(1);
       jest.advanceTimersByTime(TICK_TIME);
       await Promise.resolve();
-      expect(controller._executePoll).not.toHaveBeenCalled();
+      expect(controller._executePoll).toHaveBeenCalledTimes(1);
       jest.advanceTimersByTime(TICK_TIME);
       await Promise.resolve();
       expect(controller._executePoll).toHaveBeenCalledTimes(1);
-      jest.advanceTimersByTime(TICK_TIME * 3);
+      jest.advanceTimersByTime(TICK_TIME);
       await Promise.resolve();
       expect(controller._executePoll).toHaveBeenCalledTimes(2);
+      jest.advanceTimersByTime(TICK_TIME * 3);
```

Comment:
Is it necessary to call `_executePoll` so many times in this test? Given how we test polling in other controllers, it seems that only testing `setTimeout(0)` once and then testing `advanceTimersByTime` twice would be enough.

Reply:
done https://github.com/MetaMask/core/pull/1874/commits/1cffb112aa0201de4b16229af3ab3ecd697f8347